hail/src/main/scala/is/hail/expr/ir/EmitStream.scala

```diff
@@ -193,27 +193,43 @@ object CodeStream { self =>
       CodeStream.flatMap(map(f))
   }
 
-  def range(mb: EmitMethodBuilder[_], start: Code[Int], step: Code[Int], len: Code[Int]): Stream[Code[Int]] = {
```

Comment:
I didn't end up using iota for StreamTake, but I anticipate it being useful for other nodes, so I left it in.

Reply:
what does the name come from? This is just an infinite range stream, right?

Reply:
It's a traditional name in programming languages for the infinite range function. It started in APL, and is used in C++ and others. Happy to give it a more descriptive name.